sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -1074,21 +1074,21 @@ public partial class IncidentRootCause
         public static bool operator !=(Azure.AI.MetricsAdvisor.Models.IngestionStatusType left, Azure.AI.MetricsAdvisor.Models.IngestionStatusType right) { throw null; }
         public override string ToString() { throw null; }
     }
-    public partial class MetricAnomalyAlertConditions
-    {
-        public MetricAnomalyAlertConditions() { }
-        public Azure.AI.MetricsAdvisor.Models.MetricBoundaryCondition MetricBoundaryCondition { get { throw null; } set { } }
-        public Azure.AI.MetricsAdvisor.Models.SeverityCondition SeverityCondition { get { throw null; } set { } }
-    }
-    public partial class MetricAnomalyAlertConfiguration
+    public partial class MetricAlertConfiguration
     {
-        public MetricAnomalyAlertConfiguration(string detectionConfigurationId, Azure.AI.MetricsAdvisor.Models.MetricAnomalyAlertScope alertScope) { }
+        public MetricAlertConfiguration(string detectionConfigurationId, Azure.AI.MetricsAdvisor.Models.MetricAnomalyAlertScope alertScope) { }
         public Azure.AI.MetricsAdvisor.Models.MetricAnomalyAlertConditions AlertConditions { get { throw null; } set { } }
```

Comment:
out of curiosity, what was the reason to drop the `Anomaly` from it? and why it happens to only that type and not the others like this one?

Reply:
We're just aligning to what other languages are doing (.NET was the odd one out). We'll also rename `MetricAnomalyAlertConditions` to something else in another PR. Regarding other related models, changing them across languages is an effort with too low priority to be done in the current time frame.